src/background/services/storage.ts

```diff
@@ -1,16 +1,23 @@
-import type { Storage, StorageKey } from '@/shared/types'
+import type {
+  GrantDetails,
+  Storage,
+  StorageKey,
+  WalletAmount
+} from '@/shared/types'
 import { type Browser } from 'webextension-polyfill'
 import { EventsService } from './events'
 
 const defaultStorage = {
+  version: 2,
```

Comment:
For migrations, change this version and add a migration script in `MIGRATIONS` below.
New additions that can be dynamically set don't need migrations (e.g. we check if value is null etc.) but other structural changes would need migrations for keeping compatibility with existing installations. Also see the comment in `MIGRATIONS`.

Reply:
I'll add this comment in code.